pol.html

```diff
@@ -348,10 +348,9 @@ <h2 id="milestones">Set Conformance Milestones</h2>
 
 		<p class="totop"><a href="#contents">back to page contents</a></p>
 
-
 		<h2 id="thirdparty">Consider Third-party Content</h2>
-		<p>Procured or syndicated third-party content needs to be considered as part of an organizational policy. Where there is a separate procurement policy or process, include the accessibility policy standards and conformance levels as requirements that third-party suppliers must meet. Also consider what verification tasks, such as expert evaluation and/or user testing, should be included in the procurement process.</p>
```

Comment:
This is talking more about procurement and verification rather than third-party content. I wonder if this should be moved into its own section called something like "procurement"? Seems useful to have, just not in this section IMO.